magda-web-client/src/index.js

```diff
@@ -61,60 +28,21 @@ const store: Store = createStore(
   )
 );
 
-const recordNewRoute = location => {
-  window.ga("set", "location", document.location);
-  window.ga("send", "pageview");
-  browserHistory.lastLocation = browserHistory.currentLocation;
-  browserHistory.currentLocation = location;
-};
-recordNewRoute(browserHistory.getCurrentLocation());
-browserHistory.listen(recordNewRoute);
-
-function loadDefaultData(store) {
-  store.dispatch(requestWhoAmI());
-}
+// const recordNewRoute = location => {
```

Comment:
Should this be left commented out? Might need a TODO for it in that case :).

Reply:
it is called in the componentWillMount in the wrapper component. I should have removed these lines rather than commenting out.

Reply:
oh you need the google analytics. good point.I'll add it back